tests/examples_arguments_syntax/arrow_vector.py

```diff
@@ -0,0 +1,42 @@
+"""
+Arrow Vector
+------------------------------
+This example shows a basic vector plot use point mark ``triangle`` as shape,
```

Comment:
```suggestion
This example shows a basic vector plot using point mark ``triangle`` as shape,
```
Same change in other file